net/dnscrypt-proxy/files/dnscrypt-proxy.init

```diff
@@ -33,7 +33,7 @@ create_config_file() {
     config_get      client_key      $1 'client_key'     ''
     config_get      syslog_prefix   $1 'syslog_prefix'  'dnscrypt-proxy'
     config_get      query_log_file  $1 'query_log_file' ''
-    config_get      log_level       $1 'log_level'	''
+    config_get      log_level       $1 'log_level'	'5'
```

Comment:
You're right some parameters are missing, like LogLevel. And yes it is a good idea to share it, no problem. This is what makes open source and community software great.

I am a bit confused because dnscrypt-proxy config file specifies 6 as default log level : https://github.com/jedisct1/dnscrypt-proxy/blob/master/dnscrypt-proxy.conf#L144